doc/examples/test-examples.js

```diff
@@ -0,0 +1,34 @@
+const { readdirSync, statSync } = require('fs');
+const { join } = require('path');
+const execa = require('execa');
+const exampleDirs = readdirSync(__dirname)
+	.map(dir => join(__dirname, dir))
+	.filter(dir => statSync(dir).isDirectory());
+
+function runner(dir) {
+	let child = execa('npm install && npm test', {
```

Comment:
`execa` works a bit differently that `child_process.spawn`. This should probably be something like:

```js
function runner(dir) {
  const install = execa('npm', [ 'install' ], { cwd: dir })
  return install.then(()=> execa('npm', ['test'], { cwd: dir })
}
```

Reply:
👍 That's on me for not looking at the documentation intelligently...